resources/todomvc/vanilla-examples/javascript-es6/dist/template.js

```diff
@@ -1,20 +1,20 @@
-'use strict';
-
+/* eslint no-unused-vars: 0 */
+"use strict";
 
 const htmlEscapes = {
-    '&': '&amp',
-    '<': '&lt',
-    '>': '&gt',
-    '"': '&quot',
-    '\'': '&#x27',
-    '`': '&#x60'
+    "&": "&amp",
+    "<": "&lt",
+    ">": "&gt",
+    '"': "&quot",
```

Comment:
It seems inconsistent to use '' here. Why not just "\""?

Reply:
I ran it through prettier and eslint 🤷 
It's consistent with the other todomvc apps, but I can manually adjust it if needed.